include/odp/api/spec/packet_io.h

```diff
@@ -979,6 +981,20 @@ int odp_pktio_promisc_mode(odp_pktio_t pktio);
 int odp_pktio_mac_addr(odp_pktio_t pktio, void *mac_addr, int size);
 
 /**
+ * Set the default MAC address of a packet IO interface.
+ *
+ * @param	pktio     Packet IO handle
+ * @param[in]	mac_addr  Input buffer containing MAC address
+ * @param[in]	size      Size of the MAC address contained in
+ *			  input buffer
+ *
```

Comment:
There's no need to highlight that these params are input, so remove [in] from param tags.

Also, no need to use term "input buffer". Simply, "mac_addr  MAC address to be set as the default"